examples/snake/main.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import (
+	"fmt"
+	"image/color"
+	"log"
+	"math/rand"
+
+	"github.com/hajimehoshi/ebiten"
+	"github.com/hajimehoshi/ebiten/ebitenutil"
+	"github.com/hajimehoshi/ebiten/inpututil"
+)
+
+const (
+	screenWidth  = 640
+	screenHeight = 480
+	xNumInScreen = screenWidth / gridSize
+	yNumInScreen = screenHeight / gridSize
+	gridSize     = 10
+)
+
+const (
+	dirLeft = iota + 1
```

Comment:
`+ 1` is not needed, right?

Also, you can move these definitions into the function Update.

Reply:
`0`, I have represented as snake is `notMoving`. It will be set when user presses `Escape` or the program just starts. The snake is stationary at that moment.

Reply:
OK so let's define the values like this

```
const (
   dirNone = iota
   dirLeft
   ...
)
```

Reply:
done